lib/data-types.js

```diff
@@ -225,6 +225,59 @@ Object.defineProperty(NUMBER.prototype, 'ZEROFILL', {
   }
 });
 
+function NUMBER(options) {
```

Comment:
You're right! I think I probably left that in by accident, so I'll go ahead and take that out.